server/src/services/userService.ts

```diff
@@ -78,6 +96,8 @@ export class UserService {
         return "User not available!";
       }
 
+      const deleteCheckIn = await CheckInModel.findByIdAndDelete(id);
```

Comment:
i wouldn't put this here. remove it for now. Eventually when we remove a user we will go over everything that needs to happen.